app/src/componentLibrary/sidebar/sidebar.scss

```diff
@@ -17,24 +17,60 @@
 .sidebar-container {
   height: 100%;
   font-family: $FONT-ROBOTO-REGULAR;
+  width: 48px;
+  transition: width 0.3s ease-out;
+  transition-delay: 0.5s;
+  position: relative;
+  z-index: 1;
+  overflow: hidden;
 }
 
 .sidebar {
+  position: absolute;
+  left: 0;
+  top: 0;
   display: flex;
   flex-direction: column;
-  align-items: center;
-  position: relative;
   padding: 16px 0px 24px 0;
-  width: 48px;
   height: calc(100% - 40px);
   gap: 32px;
-  background-color: $COLOR--darkmode-gray-500;
+  background-image:
+    linear-gradient(to right, $COLOR--darkmode-gray-500 0 48px,$COLOR--darkmode-gray-450 48px 100%);
   z-index: 3;
+
+  width: 328px;
+  align-items: start;
 }
 
-.logo {
-  width: 48px;
+.open {
+  width: 328px;
```

Comment:
Do we really need this duplicated with line 41?

Reply:
These are different properties:
![image](https://github.com/reportportal/service-ui/assets/45054016/e6273ba2-be4a-4099-b5ac-029f675a4cdf)